### PR TITLE
.travis.yml: Merged commands to one line.

### LINKED ISSUE #30: Merge line 6 and 7
Line 6 and 7 could have been one, no need to add an extra line.
label: difficulty/low
Opened by @aashutoshrathi at [Gitter](https://gitter.im/coala/coala?at=59ffd0bc5a1758ed0f926903)

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Redundant line removed.
Closes #30

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,8 +3,7 @@ services: docker
 
 script:
   - >
```
Comment (jayvdb) [inline .travis.yml]: why are you using this still?

**Review by jayvdb — CHANGES_REQUESTED** [review]

**Comment by jayvdb** [comment]
Please fix your commit message

**Review by anshumanv — COMMENTED** [review]
I think commit message should go like `.travis.yml: Changed line 6 and 7 to one line` :smile:

**Comment by anshumanv** [comment]
Or `Merged line 6 and 7 to a single line`
if @jayvdb agrees, not sure tho.

**Comment by jayvdb** [comment]
'Merged ..' is better than 'Changed ..' , but the line numbers are irrelevant to why the change is occurring, and line numbers will be wrong after a few commits anyway.

But there are other more serious problems with the commit message.

**Comment by anshumanv** [comment]
> but the line numbers are irrelevant to why the change is occurring, and line numbers will be wrong after a few commits anyway.  

:+1:   
Thanks for the followup @jayvdb.  

I think @aashutoshrathi, you need to specify what commands are you merging. :smile:

**Comment by aashutoshrathi** [comment]
@jayvdb please review again

**Comment by jayvdb** [comment]
ack https://github.com/coala/rultor-python/pull/31/commits/37bfd06424fe92a6cf76d7693d58c52ec5a96cdf

**Review by jayvdb — APPROVED** [review]

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**